packages/services/api/src/modules/organization/providers/organization-members.ts

```diff
@@ -178,11 +188,12 @@ export class OrganizationMembers {
               `
             : sql``
         }
+        ${searching ? sql`AND to_tsvector("u"."display_name" || ' ' || "u"."email") @@ to_tsquery('simple', ${searchTerm + ':*'})` : sql``}
       ORDER BY
         "om"."organization_id" DESC
+        , "om"."created_at" DESC
         , "om"."user_id" DESC
-        , "om"."user_id" DESC
-      ${first ? sql`LIMIT ${first + 1}` : sql``}
+      LIMIT ${first + 1}
```

Comment:
This enforces the pagination. Whereas before it was optional. This is critical to avoid performance issues from the filter.